friskby/GW/air_in_bergen.py

```diff
@@ -49,8 +53,6 @@ def post_data(device, df):
         for i in range(len(df)):
             if df[i][component] == komponent_id[dev_id]:
                 clock = df[i][tid]
-                if clock.strip() == '00:00':
-                    clock = '23:59'
                 ts = current_date + "T" + clock + ":00Z"
```

Comment:
If I understand this correctly, we will still get an incorrect timestamp as `clock` is the time in the local timezone but inserted in a timestring indicating Zulu time with the `Z`. Zulu time equals UTC.